tests/framework/installer/ceph_installer.go

```diff
@@ -127,13 +129,26 @@ func (h *CephInstaller) CreateCephOperator(namespace string) (err error) {
 		h.k8shelper.ChangeHostnames()
 	}
 
-	rookOperator := h.Manifests.GetRookOperator(namespace)
+	rookOperator := h.Manifests.GetRookOperator(namespace, h.EnableCSI)
 
 	_, err = h.k8shelper.KubectlWithStdin(rookOperator, createFromStdinArgs...)
 	if err != nil {
 		return fmt.Errorf("Failed to create rook-operator pod : %v ", err)
 	}
-
+	if h.EnableCSI {
+		csiTemplate := h.Manifests.GetCephCSI(namespace)
```

Comment:
Why does `GetCephCSI()` need to be separate from `GetRookOperator`? To think about how the example yamls will be, we will have common.yaml including all the CSI RBAC changes, and operator.yaml will include all the CSI settings on the operator. It's just part of the operator, and no extra steps. This separation makes it look like there is an extra step.